docs/xostor/xostor.md

```diff
@@ -520,6 +592,95 @@ Consider the following:
 - If you create a volume of 200 GiB it means that 400 of the 600 GiB are physically used. However, the remaining disk of 200 GiB cannot be used because it cannot be replicated on two different disks.
 - If you create 3 volumes of 100 GiB: the SR is filled. In this case, you have 300 GiB of unique data and a replication of 300 GiB.
 
+### How to add a new host or fix a bad configured host?
+
+:::warning
+For a new host, make sure you have the required packages by running these commands on it:
+```
+yum install -y xcp-ng-release-linstor
+yum install -y xcp-ng-linstor
```

Comment:
This could bring them versions that are newer than the installed versions; So first they need to make sure their pool is up to date. But in some cases they need the new host before they can update the rest (for example so that they have enough hosts for all VMs during RPU), so we need to provide instructions for that. There's a KB in the general, non-linstor case, we should work together on a doc update (or give thomas all the data and let him write it), covering both the standard case and the linstor case.

Something in another section of the docs named for example "Prepare a new host for joining an existing pool"

Reply:
Until then, you could say here that the pool must be up to date, and if it isn't and they aren't in a situation where they can safely update it, they should first contact support.